@ecomplus/storefront-template/template/scss/themes/ecom-beauty/_custom.scss

```diff
@@ -0,0 +1,156 @@
+@import url('https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap');
+@import "header";
+@import "product-card";
+@import "product";
+@import "footer";
+
+body {
+  padding-bottom: 40px!important;
+}
+
+main {
+  .top-bar {
+    background-color: $body-bg;
+
+    &__nav {
+      color: $body-color-secondary;
+    }
+  }
+
+  .h1, .h2, .h5 {
+    font-family: 'Playfair Display';
+    font-style: normal;
+    font-weight: normal;
+    letter-spacing: 0;
+    color: $secondary;
+  }
+  .h1 {
+    font-size: 40px;
+    line-height: 42px;
+  }
+
+  .h2 {
+    font-size: 32px;
+    line-height: 36px;
+  }
+
+  .product__name, .h4, .h6 {
+    font-family: 'Poppins';
+    font-style: normal;
+    font-weight: 600;
+    letter-spacing: 0;
+    color: $secondary;
+  }
+
+  .h4 {
+    font-size: 24px;
+    line-height: 29px;
+  }
+
+  .h5 {
+    font-size: 18px;
+    line-height: 22px;
+  }
+
+  .h6 {
+    font-size: 16px;
+    line-height: 19px;
+  }
```

Comment:
Isso é pra onde no caso? A intenção é realmente sempre que for h4, h5, h6?
Se forem elementos específicos acho que seria o caso de usar seletores melhores, não utility classes...